site/content/entry/yuzu-progress-report-nov-2021/index.md

```diff
@@ -0,0 +1,326 @@
+both games need other formats that are still missing and some other fixes.
+
+Turns out, while [there is support for both OpenGL and Vulkan](https://github.com/yuzu-emu/yuzu/pull/7357) now, only NVIDIA declares support on Vulkan, meaning that any 
+affected game running on an AMD or Intel GPU will need to stick to OpenGL to get proper rendering.
+
+{{< single-title-imgs
+    "Testing homebrew"
+    "./s8uintbug.png"
+    "./s8uintfix.png"
+    >}}
+
+[vonchenplus](https://github.com/vonchenplus) found a bug where image creation would include width size restrictions that wouldn’t apply to later updates and downloads, 
+making OpenGL spam errors.
+[Properly handling those cases](https://github.com/yuzu-emu/yuzu/pull/7294) lets the driver calm down.
+Proper and cleaner code can also mean less driver overhead.
+
+Continuing with the implementation of unsupported formats, vonchenplus added `R5G6B5_UNORM_PACK16` [textures to Vulkan](https://github.com/yuzu-emu/yuzu/pull/7291) and 
+`S8_UINT_D24_UNORM` [globally](https://github.com/yuzu-emu/yuzu/pull/7466). 
+Again, just like S8_UINT, AMD and Intel hardware lack Vulkan support for S8_UINT_D24_UNORM.
+
+vonchenplus also [converted all usages of legacy attributes](https://github.com/yuzu-emu/yuzu/pull/7375) within shaders into generic attributes, since they were not 
+supported by some graphics drivers, improving compatibility.
+
+These four changes help `DRAGON QUEST BUILDERS`, but more work is required to make the game playable. If you're interested in testing this game, use Vulkan, or 
+disable Asynchronous GPU Emulation when using OpenGL.
+Beware, crashes may happen.
+
+{{< imgs
+	"./dqb.png| Missing graphics make the game not an optimal experience for now (DRAGON QUEST BUILDERS)"
+  >}}
+
+Back to fixes to the changes introduced by the resolution scaler, users reported that taking screenshots at 1x resolution multiplier resulted in saved images with a size 
+higher than the rendering resolution of the game.
+A mix up resulted in the screenshot being saved at the window size instead of the game’s framebuffer size.
+epicboy [fixed the logic](https://github.com/yuzu-emu/yuzu/pull/7389) and we’re back to proper screenshot resolutions.
+
+[liushuyu](https://github.com/liushuyu), our fantastic Webadmin, got reports of multiple GPU systems having crashes while running the Flatpak version of yuzu on Linux.
+Seems like the bundled version of mesa and ffmpeg will conflict on systems with different GPU vendors. Just the usual weird GNOME decisions.
+
+liushuyu submitted [two](https://github.com/yuzu-emu/yuzu/pull/7431) [separate](https://github.com/yuzu-emu/yuzu/pull/7467) pull requests addressing this problem, but more 
+edge cases continue to appear, so the battle isn’t over yet.
+
+While we continue to work to win this war against Flatpak, please stick to using our appimages, or [manually build yuzu](https://yuzu-emu.org/wiki/building-for-linux/) if 
+you prefer.
```

Comment:
```suggestion
While we continue to work to win this war against Flatpak, please stick to using our [AppImages](https://yuzu-emu.org/downloads/#linux), or [manually build yuzu](https://yuzu-emu.org/wiki/building-for-linux/) if 
you prefer.
```

You probably want to add that if manually building yuzu from the source code, the user is responsible to make sure the ffmpeg libraries in their system are capable of hardware decoding or at least not broken and make sure they support all of VP8, VP9, and H264 codecs.